web/modules/custom/dbcdk_community_content/src/FieldNormalizer/ImageItemFieldNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @file
+ * Contains ImageItemFieldNormalizer.
+ */
+
+namespace Drupal\dbcdk_community_content\FieldNormalizer;
+
+use Drupal\Core\Field\FieldItemBase;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Normalize the output for an Image field.
+ */
+class ImageItemFieldNormalizer implements FieldNormalizerInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize(FieldItemBase $field) {
+    // Load the file by target_id.
+    /* @var \Drupal\file\Entity\File $image */
+    $image = File::load($field->get('target_id')->getString());
```

Comment:
I have not been able to find any. This seems like a similar case to when we have to create links and use Link::createFromRoute() (just one example of Link/Url).